tests/test_verifylib.py

```diff
@@ -1326,7 +1354,119 @@ def tearDownClass(self):
   def test_verify_demo_as_sublayout(self):
     """Test super layout's passing sublayout verification. """
     verify_sublayouts(
-        self.super_layout, self.super_layout_links)
+        self.super_layout, self.super_layout_links, ".")
+
+
+
+
+
```

Comment:
whitespace?

Reply:
That's what the [style guideline suggests](https://github.com/secure-systems-lab/code-style-guidelines#blank-lines). I know, in a recent comment I said that I prefer 3 over 5 blank lines. But I guess I felt particularly guideline obedient here. :)

Should we open a ticket to clean up all blank lines everywhere?

Reply:
nah, I don't think it's necessary